server/middleware/authencicate.js

```diff
@@ -0,0 +1,14 @@
+const jwt = require('jsonwebtoken')
+
+const authencicate = function (req,res,next) {
+    try{
+        const token = req.headers.token
+        const decoded = jwt.verify(token,process.env.SECRET)
```

Comment:
JWT definisiin di helpers
```suggestion
const verify = require('../helpers/jwt').verify

const authencicate = function (req,res,next) {
    try{
        const token = req.headers.token
        const decoded = verify(token,process.env.SECRET)
```